src/mavesp8266.h

```diff
@@ -62,7 +63,7 @@ class MavESP8266GCS;
 //-- TODO: This needs to come from the build system
 #define MAVESP8266_VERSION_MAJOR    1
 #define MAVESP8266_VERSION_MINOR    0
-#define MAVESP8266_VERSION_BUILD    8
+#define MAVESP8266_VERSION_BUILD    9
```

Comment:
Eventually I would want to have the build system updating this automatically. But in the mean time, we need to manually bump the build number each time.